packages/bot-skeleton/src/utils/error-handling.js

```diff
@@ -0,0 +1,33 @@
+import { localize } from '@deriv/translations';
+import { error_message_map } from './error-config';
+
+const onKeyDownHandleError = observer => {
+    if (Blockly.selected !== null) {
+        //keyboard gives keyCode 46 and Laptop gives 8
+        if (event.keyCode === 8 || event.keyCode === 46) {
```

Comment:
Can you please check if we can generalize this condition with if (event.key === 'Delete' || event.keyCode === 46)

Reply:
done ✅